packages/strapi-plugin-sentry/services/__tests__/sentry.test.js

```diff
@@ -0,0 +1,104 @@
+'use strict';
+
+const INVALID_DSN = 'an_invalid_dsn';
+const VALID_DSN = 'a_valid_dsn';
+const captureException = jest.fn();
+
+jest.mock('@sentry/node', () => {
+  return {
+    init(options = {}) {
+      if (options.dsn !== VALID_DSN) {
+        throw Error();
+      }
+    },
+    captureException,
+    withScope(configureScope) {
+      configureScope();
+    },
+  };
+});
+
+let sentryService = require('../sentry');
+const defaultConfig = require('../../config/settings.json');
+
+describe('strapi-plugin-sentry service', () => {
+  beforeEach(() => {
+    // Reset Strapi state
+    global.strapi = {
+      config: {},
+      plugins: {
+        sentry: {
+          config: defaultConfig,
+        },
+      },
+      log: {
+        warn: jest.fn(),
+        info: jest.fn(),
+      },
+    };
+    // Reset the plugin resource state
+    jest.resetModules();
```

Comment:
On a very nitpicky note, I would suggest to move this line in an `afterEach` hook so that it feels more like "you're cleaning after leaving the room" instead of "cleaning before entering the room". 😊

Reply:
Done, thanks